services/api-server/src/simcore_service_api_server/api/routes/licensed_items.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends, status
+
+from ...api.dependencies.authentication import get_product_name
+from ...api.dependencies.webserver_rpc import get_wb_api_rpc_client
+from ...exceptions.service_errors_utils import DEFAULT_BACKEND_SERVICE_STATUS_CODES
+from ...models.pagination import Page, PaginationParams
+from ...models.schemas.model_adapter import LicensedItemGet
+from ...services_rpc.wb_api_server import WbApiRpcClient
+
+router = APIRouter()
+
+_LICENSE_ITEMS_STATUS_CODES: dict[int | str, dict[str, Any]] = {
+    **DEFAULT_BACKEND_SERVICE_STATUS_CODES,
+}
+
+
+@router.get(
+    "/page",
```

Comment:
we used `/page` only for old collections in order to keep backwards compatibility with the list operation
List operations (from now on) should all incorporate pagination i.e. if `GET books/` I expect to respond with `Page[BookGet]`

therefore here i would drop `/page`

Reply:
Yep, good point. Done